src/Routing/RecurlyRouteSubscriber.php

```diff
@@ -119,12 +118,11 @@ protected function alterRoutes(RouteCollection $collection) {
           array(
             '_controller' => '\Drupal\recurly\Controller\RecurlySubscriptionSelectPlanController::planSelect',
             '_title' => \Drupal::config('recurly.settings')->get('recurly_subscription_max') == 1 ? 'Signup' : 'Add plan',
-            'operation' => 'select_plan',
           ),
           array(
             '_entity_access' => "$entity_type_id.update",
             '_access_check_recurly_user' => 'TRUE',
-            '_access_check_recurly_default' => 'TRUE',
```

Comment:
@markdorison I added the local account access check. However there were a few access checks where we would not want that, like sign up for example.